### ISSUE TITLE
3001 HTTPS support

### ISSUE THREAD

**Issue by fboucquez** [symbol/symbol-explorer#927]
Atm I can see that there is HTTP and 3000 hardcoded when resolving the gateway URL address of a node. 

https://github.com/symbol/symbol-explorer/blob/dev/src/infrastructure/NodeService.js#L117

The idea is to convert all our nodes (and community eventually) to https port 3001. Is it possible to update the code? Ideally, the URL resolution should "ping" 3001 and 3000 to know if the node has HTTPS support yet.

Note that this is different from the explorer configuration where we should be able to update without a code change.

https://github.com/symbol/symbol-explorer/blob/dev/src/config/default.json#L5

We should revisit the statistics service for a similar gateway URL resolution from a node's hostname

**Comment by fboucquez** [comment]
About the exception handling, you can see the problem by going to this HTTP node:

https://explorer.rockbear.io/nodes/965C7555721D1503F5024E02F42979FD6833142A63483CE3AB5E35F7EB094432

![image](https://user-images.githubusercontent.com/5390558/129761218-dd89b41c-7580-4350-9b18-834e51ea70a3.png)

**Comment by AnthonyLaw** [comment]
> if a HTTPS explorer tries to talk to a HTTP node, the browser would reject the call.

not sure it's a good idea, because even nodes without HTTPS enabled, still consider valid nodes.

**Comment by rg911** [comment]
would be better to show both http and https nodes, can probably add a `protocol` column next to the port in node list?

**Comment by fboucquez** [comment]
Yes, we should show both http and https nodes. The problem would be that when asking a HTTP only node for the health and info, the browser will reject. HTTPS explorer cannot talk to a HTTP node for the state. See the screenshot above

**Comment by AnthonyLaw** [comment]
> Yes, we should show both http and https nodes. The problem would be that when asking a HTTP only node for the health and info, the browser will reject. HTTPS explorer cannot talk to a HTTP node for the state. See the screenshot above

alright, got it now 👍🏼

**Comment by AnthonyLaw** [comment]
> would be better to show both http and https nodes, can probably add a `protocol` column next to the port in node list?

yeah, but I think it should be test and store into the static service.
